metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -513,9 +520,16 @@ def _get_dataset_column_min(
             logger.debug(
                 f"Caught exception while attempting to get column min for column {column}. {e}"
             )
+
             self.report.report_warning(
-                "Profiling - Unable to get column min",
-                f"{self.dataset_name}.{column}",
+                title="Unable to Calculate Min",
+                message="The min for the column will not be accessible",
+                context=f"""
+                    {{
+                        "column": "{self.dataset_name}.{column}",
+                        "errorMessage": "{e}"
+                    }}
```

Comment:
use the `exc` parameter for the exception. the context should only have the dataset + column names